hub/core/storage/s3.py

```diff
@@ -83,6 +85,7 @@ def __setitem__(self, path, content):
         except Exception as err:
             raise S3SetError(err)
 
+    @retry(stop_max_attempt_number=3)
```

Comment:
new depend :(

Reply:
boto config accepts a retry arg, whats wrong with that?

Reply:
oh nice i didn't know that